client/tests/webdriver/baseSpecs/assessmentsForTasks.spec.js

```diff
@@ -87,6 +87,21 @@ describe("For the periodic task assessments", () => {
         "monthly",
         ADVISOR_1_TASK_EDIT_DETAILS
       )
+    })
+
+    it("Should allow seeing add button for the assessments in the future", async() => {
+      await (
+        await ShowTask.getNextPeriodButton("subTaskMonthly", "monthly")
+      ).waitForExist()
+
+      await (
+        await ShowTask.getNextPeriodButton("subTaskMonthly", "monthly")
+      ).click()
+
+      await (
+        await ShowTask.getFutureAddAssessmentButton("subTaskMonthly", "monthly")
```

Comment:
You are testing this against subTaskMonthly, yet you have not set `allowFutureAssessments: true` for that assessment definition.